src/components/Element.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {addItem} from "../actions/favGifs";
+import {connect} from "react-redux";
+import {bindActionCreators} from "redux";
+import {LOCAL_STORAGE_FAVS} from '../middleware/favStorage';
+import {
+    RelativeDiv,
+    ElementImg,
+    IconFav,
+    IconFavBlue,
+    Shadow
+} from './Element.styled';
+
+const Element = (props) => {
+    const localStorageFavsId = JSON.parse(localStorage.getItem(LOCAL_STORAGE_FAVS));
+
+    if (!localStorageFavsId || !localStorageFavsId[props.gif.id]) {
+        return (
+            <RelativeDiv onClick={() => props.addItem(props.gif)}>
+                <Shadow>
+                    <IconFav className="fas fa-thumbs-up fa-3x"></IconFav>
```

Comment:
why are you using className? you shouldn't need classNames with styled components